### ISSUE TITLE
Expected versions of engines.node should be an even numbered release

### ISSUE THREAD

**Issue by wrumsby** [bigskysoftware/htmx#1630]
As per #1244 and the existing `package.json` `engines.node` expects Node 15.x.

```json
  "engines": {
    "node": "15.x"
  }
```

Odd numbered versions of Node are experimental and not recommended for production use. While the usage here is installing `htmx.org` for use with a bundler, meaning you're theoretically only running your build with "non production" software, it can create issues with other dependencies if you have paranoid npm settings (i.e. `engine-strict=true`).

TL;DR could we change `engines.node` to `14.x || 15.x` or similar? Happy to make a PR if that helps.

**Comment by ralyodio** [comment]
agreed, odd are alpha. and node 15 is deprecated anyway

**Comment by alexpetros** [comment]
Closing as a duplicate of https://github.com/bigskysoftware/htmx/issues/1584

But yes, that bugs me too. It only affects htmx developers, but eventually I'll get that upgraded.
